core/lsp/helpers.h

```diff
@@ -9,7 +9,8 @@ core::TypePtr getResultType(const core::GlobalState &gs, const core::TypePtr &ty
                             core::TypePtr receiver, const core::TypeConstraint *constr);
 
 std::string prettyTypeForMethod(const core::GlobalState &gs, core::MethodRef method, const core::TypePtr &receiver,
-                                const core::TypePtr &retType, const core::TypeConstraint *constraint);
+                                const core::TypePtr &retType, const core::TypeConstraint *constraint,
+                                const bool concretizeIfAbstract = false);
```

Comment:
In combination with the ShowOptions change above, I think it would likely be better if we made the `options` argument not have a default. There are few enough calls to these methods that it wouldn't hurt to explicitly create a `ShowOptions` struct instance and pass it at all the call sites, just so people are aware what they're dealing with.

Reply:
This was also fixed in 0849d83.